src/Apps/Order/Routes/Shipping2/Components/CollapseDetails.tsx

```diff
@@ -0,0 +1,19 @@
+import { Collapse } from "@artsy/palette"
+import { useShippingContext } from "Apps/Order/Routes/Shipping2/Hooks/useShippingContext"
+import React from "react"
+
+export const CollapseDetails: React.FC = ({ children }) => {
+  const shippingContext = useShippingContext()
+
+  const showArtsyShipping =
+    shippingContext.state.stage === "shipping_quotes" &&
+    !!shippingContext.orderData.savedFulfillmentDetails?.isArtsyShipping &&
+    !!shippingContext.orderData.shippingQuotes &&
+    shippingContext.orderData.shippingQuotes.length > 0
+
+  return (
+    <Collapse data-testid="ShippingQuotes_collapse" open={showArtsyShipping}>
+      {children}
+    </Collapse>
+  )
```

Comment:
Now that we have a context, we can colocate behavior in self-describing components, and reduce the amount of cognitive, logical overhead in parent container components. Also, this allows us to reduce props passing.